### ISSUE TITLE
English version?

### ISSUE THREAD

**Issue by JKwapinski** [rupikon/medicine-warriors#9]
There can be people who can't read cyrillic, so maybe english version should be provided for them?

**Comment by mroma9000** [comment]
From testing: the site has a headline translations under each section but there are no translations for descriptions available. (Could potentially confuse)

**Comment by piotr-konorski** [comment]
That's right! As we focused mainly on Ukraine for now, we still didn't have time to prepare a switch to other languages.. I will add it as a new feature to 'frontend' project! Thanks

**Comment by piotr-konorski** [comment]
@NewstellerBot could You please prepare a template for switching between languages, then we will together with Olha prepare translated text :)
